companion/src/firmwares/opentx/opentxinterface.h

```diff
@@ -116,6 +116,7 @@ class OpenTxFirmware: public Firmware
       addLanguage("se");
       addLanguage("sk");
       addLanguage("tw");
+      addLanguage("ua");
```

Comment:
ISO 639-1 language code for Ukrainian is `uk` no `ua`.

https://en.wikipedia.org/wiki/Ukrainian_language

Reply:
That is correct, but not of relevance at present as you'll see from the TODO line a few lines up ;)